src/compiler/_namespaces/ts.ts

```diff
@@ -72,3 +71,5 @@ import * as moduleSpecifiers from "./ts.moduleSpecifiers";
 export { moduleSpecifiers };
 import * as performance from "./ts.performance";
 export { performance };
+/** @internal */ export * as NodeConstructors from "../nodeConstructors";
+/** @internal */ export * as ObjectConstructors from "../objectConstructors";

```

Comment:
There's no linter or note for this (I should do so), but the more verbose form is preferred as it generates better output in esbuild, thanks to a bug I have yet to minimize.